### ISSUE TITLE
[BUG] Custom Commands Running Bash Scripts with "${file}" fail silently when file path contains a space and "Wait for Exit" is set to False.

### ISSUE THREAD

**Issue by PConrad-NI** [ForkIssues/TrackerWin#2367]
Running the latest version of Fork on Win 11.

I have a simple custom file command set up as a bash script...

`echo "${file}" > /c/temp/debug.txt 2>&1`

![image](https://github.com/user-attachments/assets/06980cf6-c1f2-4a7e-b5c7-7634031d5b1d)

![image](https://github.com/user-attachments/assets/f541f9ed-9667-45f6-9b2f-632b97d4f758)

I'm currently working out of a .fork directory and I have the following setup.

![image](https://github.com/user-attachments/assets/c32c706e-f88f-44f3-b7ba-75a2ad12c955)

If `Wait for Exit` is checked, running my custom command on any file yields proper expansion of ${file}

Debug file contents A: `.fork/no-space-test/baz.txt`
Debug file contents B: `.fork/space test/bar.txt`

If `Wait for Exit` is not checked, running my custom command on any file with a path that has spaces will NOT yield proper expansion of ${file} and the script just dies silently without doing anything.

Debug file contents A: `.fork/no-space-test/baz.txt`
Debug file contents B:  !!! FILE IS NOT CREATED !!!

(This bug was very difficult for me to track down. I was interested in it because I was trying to find a way to get around some other issues such as trying to launch a script silently.)

**Comment by DanPristupov** [comment]
Yes, you are right. Good catch.

I fixed the problem and made a test build. Can you check if everything works properly now?

https://cdn.fork.dev/prerelease/Fork-2.2.12.zip

Extract the archive somewhere and run `fork.exe`.

**Comment by PConrad-NI** [comment]
Yes! It's fixed! Tested it with a longer path with multiple spaces.

`.fork/New folder/New folder/New Text Document.txt`

I am impressed with how fast you squashed this.
